marketplace/manifest.yaml

```diff
@@ -1,11 +1,11 @@
-applicationName: com.spinkube.spinkube
-publisher: "Spinkube"
+applicationName: dev.spinkube.spinkube
```

Comment:
I'm not sure on the naming here.  Perhaps the [example manifest.yaml](https://github.com/Azure-Samples/kubernetes-offer-samples/blob/main/samples/k8s-offer-azure-vote/manifest.yaml) follows the `<domain>.<offering>` pattern more to ensure a unique bundle name?  I'm assuming when the SpinKube marketplace offering is created/listed, this won't be used; rather `SpinKube` will be the offering name.

Reply:
Yeah this aligns with my thoughts after reading the example file you referenced.